src/Sylius/Component/Core/Model/AdminUser.php

```diff
@@ -80,4 +83,35 @@ public function setLocaleCode(?string $code): void
     {
         $this->localeCode = $code;
     }
+
+    public function getAvatar(): ?ImageInterface
+    {
+        return $this->avatar;
+    }
+
+    public function setAvatar(?ImageInterface $avatar): void
+    {
+        $avatar->setOwner($this);
+        $this->avatar = $avatar;
+    }
+
+    public function addAvatar(ImageInterface $avatar): void
```

Comment:
`setAvatar` and `getAvatar` should be enough because an admin can have only one avatar.